### DISCUSSION TITLE
Hi. I need your help please 🙏🏻

### DISCUSSION THREAD

**Discussion by Mayaema23** [ExistentialAudio/BlackHole#802] [Q&A]
I need for my Cubase track to go through my zoom meeting and I don’t have a loop back on my tascam 102i interface so I downloaded this . Can someone please explain to me How can I make my track go through the zoom meeting? I’m using Mac. I just don’t understand how to do it and it will be so helpful🙏🏻🙏🏻🙏🏻

**Comment by devinroth** [comment]
If you need something easy use https://dipper.audio.

Otherwise make an aggregate with BlackHole and your Mic and Speakers. Set that as IO in Cubase. Use BlackHole as input in Zoom. Do all relevant routing in Cubase. Don’t forget to pass your mic to BlackHole so they can hear you on Zoom.

Devin

**Reply by Mayaema23** [reply]
Hi thank you so much for the reply ❤️ my mic is going through my interface. Nothing to do with the mac mic. I did an aggregate between: zoom, cubase and blackhole. This will not work?

**Comment by devinroth** [comment]
If you want people to hear you on Zoom you need to mix your mic into BlackHole.

Devin

**Comment by Mayaema23** [comment]
People actually heard me through zoom the way i did it great. The problem is that they can’t here my track when I share screen. The mic is working perfectly. This is not meant for interface without loopback? 

The app you sent me in the beginning did not work either 😢

**Comment by devinroth** [comment]
Did you change the input in Zoom?

Devin

**Comment by Mayaema23** [comment]
Yes😞

בתאריך יום ד׳, 11 בספט׳ 2024 ב-19:40 מאת Devin Roth <
***@***.***>:

> Did you change the input in Zoom?
>
> Devin

**Comment by devinroth** [comment]
Send screenshots.

Devin
